awsenergylabelerlib/awsenergylabelerlib.py

```diff
@@ -119,6 +127,27 @@ def validate_accounts(accounts_):
                                              f' {accounts}')
         return accounts
 
+    @staticmethod
+    def _validate_regions(regions):
+
+        def validate_region(region):
```

Comment:
This is very loose validation and is not very safe. 

```
def get_available_regions():
    """The regions that security hub can be active in.

    Returns:
        regions (list): A list of strings of the regions that security hub can be active in.

    """
    url = 'https://api.regional-table.region-services.aws.a2z.com/index.json'
    response = requests.get(url)
    if not response.ok:
        LOGGER.error('Failed to retrieve the info')
        return []
    return [entry.get('id', '').split(':')[1]
            for entry in response.json().get('prices')
            if entry.get('id').startswith('securityhub')]
            ```
The above will retrieve the actual regions that security hub exists on.
Then some simple set math would do.